### PR TITLE
Remind users to acknowledge CC

### PR DESCRIPTION
**Pull request by breuleux**
* Update the Compute Canada link to point to the new website (Alliance Canada).
* Point users to the "Acknowledging the Alliance" page and remind them they should do so for their papers (this can help us with our allocation requests).

I inserted the reminder at the beginning because it sounded natural there, but there may be better spots to place it.

### CODE REVIEW

**Review by btravouillon — CHANGES_REQUESTED** [review]

File: docs/Extra_compute_cc.rst [hunk 1]
```diff
@@ -7,9 +7,11 @@ Compute Canada Clusters
 
 The clusters named `Beluga`, `Cedar`, `Graham`, `Narval` and `Niagara` are
 clusters provided by the `Compute Canada organisation
```
[the remark this answers is not in the document]
Comment (breuleux) [reply docs/Extra_compute_cc.rst:9]: Should we change it everywhere else in the file?

[the remark this answers is not in the document]
Comment (btravouillon) [reply docs/Extra_compute_cc.rst:9]: Nice catch. I guess yes!

[the remark this answers is not in the document]
Comment (btravouillon) [reply docs/Extra_compute_cc.rst:9]: I opened https://github.com/mila-iqia/mila-docs/issues/132 to handle this in another PR.

**Review by btravouillon — APPROVED** [review]
Looks good to me. 👍

The CC -> DRAC replacement will be handled in a separate PR.

**Review by satyaog — APPROVED** [review]

**Pull request merged**